governance/charter.md

```diff
@@ -0,0 +1,130 @@
+      * What are effective practices for implementing policy controls?
+      * How can we test, validate, explain, audit our systems?
+      * What additional measures are needed, specific to cloud, in highly regulated environments?
+  * Personas and use cases
+  * Common vocabulary to talk about and understand cloud native security
+  * CNCF project ecosystem & landscape
+  * Define security scenarios (e.g. network configuration, application security, service orchestration)
+  * Block architecture(s) for secure access
+  * Highlight trade-offs (e.g. Expressability vs Explainability)
+  * Best practices and anti-patterns (potentially highlighing where there is disagreement on these)
+* Security assessments of specific proposals or projects
+* Identify projects for consideration for CNCF
+* Cross-pollinate knowledge by participating and inviting people from other projects and SIGs to share security practices
+* Integrate relevant external standards, such as from CII or NIST, as part of educational resoruces and/or SIG processes
+
+Given that the group is comprised of volunteers, specific requests from the TOC
+may be queued according to the bandwidth of the group. The co-chairs will
+facilitate prioritization under the guidance of the SIG-Security CNCF TOC leads.
+
+### Out of scope
+* Not a standards body: We won't be creating standards.
+* Not an umbrella organization: We interact with other groups for knowledge
+  sharing, not decision-making.
+* Not a compliance body
+* Not a certification board for security of individual projects
+* We will not
+  * answer any specific questions regarding the state of security of any project
+    or product
+  * consider device security unless there is some impact to cloud systems.
+  * explore trust and safety concerns that are not specific to cloud
+    (e.g. fraud detection, user generated content moderation, spam filtering,
+    phishing, cross-site scripting attacks, SQL injection, etc.)
+* We will not ensure the saftey of any operational system.
+* This is not related to vulnerability detection and handling any specific
+  security vulnerability or attack.
+
+## Governance
+
+Security must be addressed at all levels of the stack and across the whole
+ecosystem, so the group seeks to encourage participation and membership across
+a wide range of roles, from diverse companies and organizations.
+
+### Cross-group relationships
+To focus our efforts, we avoid duplication by developing relationships with
+other groups that
+focus on a particular technology (such as Kubernetes SIGs) or have a broader
+mandate (such as government organizations).
+
+As a guide to visitors, we maintain the list of groups in the repo
+[README](https://github.com/cncf/sig-security#related-groups).
+
+Co-chairs are responsible to ensure periodic cross-group knowledge sharing,
+which is accomplished by cross-group membership, invitation to present at
+a SIG meeting and/or offering to present to the related group.
+
+## Operations
```

Comment:
I think this section duplicates the first part of the governance README, I'd remove for ease of maintenance

Reply:
removed duplicate content and left breadcrumb to README, in case people follow a link to charter and want to learn more about the governance details